### ISSUE TITLE
[Request] Add UI for creating new game resources page

### ISSUE THREAD

**Issue by MemoryTAS** [TASVideos/tasvideos#848]
Using a URL or an internal link to make a page DOES work, but it's kinda clunky and not necessarily intuitive. A proper UI would be greatly appreciated.

**Comment by adelikat** [comment]
I'm thinking a button on the game page, if there isn't already a GR page, a Create Game Resources Page displays

**Comment by adelikat** [comment]
So I ran into an issue with this idea.  Games no longer have Systems, Versions do.  But we want GR pages for a game, not a version

**Comment by MemoryTAS** [comment]
This also gets awkward in general with multiplatform games, don't think we ever really considered that. Hrm.

**Comment by MemoryTAS** [comment]
> Would moving all the GameResource page as a subpage of the Game pages be too much? It certainly makes a bit more sense to have the resource of a game under the url of the game.

I like this idea in theory, but some game resources pages are of multiple games of a series rather than individual entries. I'm not even talking pokemon versions, for example https://tasvideos.org/GameResources/SNES/MegaManX

I'm not sure how you'd handle those at this point.

**Comment by adelikat** [comment]
SMB is another good example, look how many actual games the page is tied to: https://tasvideos.org/GameResources/NES/SuperMarioBros

But I think we should do something like this.  Right now you can look for games, and this is one navigation system, from which you can get to GR pages.  And then you have a whole other nav for GR pages, which can link to games.  There is no way this is the optimal strategy.

**Comment by MemoryTAS** [comment]
I wonder if it could be intuitive to attach a GR to a game group instead of a single game entry for cases like smb or mm. Since it would be the minority, you wpuld wan

**Comment by adelikat** [comment]
Yes, you can link many games to a single GR page.  You cannot do the reverse (currently) and have many GR pages for the same game.

You also cannot currently link a group to a GR page
